decidim-meetings/app/commands/decidim/meetings/admin/create_meeting.rb

```diff
@@ -47,7 +47,8 @@ def create_meeting!
             transparent: @form.transparent,
             organizer: @form.organizer,
             registration_terms: @form.current_component.settings.default_registration_terms,
-            component: @form.current_component
+            component: @form.current_component,
+            questionnaire: Decidim::Forms::Questionnaire.new
```

Comment:
Why do we need an empty questionnaire?

Reply:
The `decidim-forms` doesn't have any code to let users create a `questionnaire`, only to edit/update it, so we are attaching a questionnaire to the meeting, like we do when a survey is created (although possibly is not the very same case).